src/test/java/teammates/test/BaseTestCaseWithLocalPostgresSqlDatabaseAccess.java

```diff
@@ -0,0 +1,48 @@
+package teammates.test;
+
+import com.zaxxer.hikari.HikariConfig;
+import com.zaxxer.hikari.HikariDataSource;
+import org.testcontainers.containers.JdbcDatabaseContainer;
+import org.testcontainers.containers.PostgreSQLContainer;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import teammates.common.datatransfer.sqlattributes.CourseAttributes;
+import teammates.common.util.HibernateUtil;
+import teammates.logic.api.LogicNewExtension;
+
+import javax.sql.DataSource;
+
+public abstract class BaseTestCaseWithLocalPostgresSqlDatabaseAccess extends BaseTestCaseWithPostgresSqlDatabaseAccess {
```

Comment:
```suggestion
public abstract class BaseTestCaseWithLocalSqlDatabaseAccess extends BaseTestCaseWithSqlDatabaseAccess {
```
It would be good for the class names to be database agnostic.